### PR TITLE
UDP services not supported with Kuryr SDN edit

### PR DESCRIPTION
**Pull request by maxwelldb**
Edit for #14680 

Need a review from @openshift/team-documentation

### CODE REVIEW

File: install_config/topics/ocp_osp_provisioning.adoc [hunk 1]
```diff
@@ -223,6 +223,13 @@ Network policies and nodeport services are not supported when Kuryr SDN is
 enabled.
 ====
 
+[NOTE]
+====
+If Kuryr SDN is enabled, OpenShift services are implemented through Octavia Amphra VMs.
```
Comment (kalexand-rh) [inline install_config/topics/ocp_osp_provisioning.adoc]: s/OpenShift/{product-title} 
^ always and forever

File: install_config/topics/ocp_osp_provisioning.adoc [hunk 2]
```diff
@@ -223,6 +223,13 @@ Network policies and nodeport services are not supported when Kuryr SDN is
 enabled.
 ====
 
+[NOTE]
+====
+If Kuryr SDN is enabled, OpenShift services are implemented through Octavia Amphra VMs. 
+
+OSP Octavia does not support UDP load balancing. Services that expose UDP ports are not supported.
```
Comment (kalexand-rh) [inline install_config/topics/ocp_osp_provisioning.adoc]: Define the TLAs if this is their first use in the file.

**Review by kalexand-rh — COMMENTED** [review]
One change and a question.
Please squash after you address them.

**Comment by maxwelldb** [comment]
@kalexand-rh Copacetic?

**Comment by kalexand-rh** [comment]
LGTM. Merging.

**Comment by kalexand-rh** [comment]
The webhook isn't working, but the build passed: https://travis-ci.com/openshift/openshift-docs/builds/112817072

**Comment by kalexand-rh** [comment]
/cherrypick-enterprise-3.11

**Pull request merged**